corehq/apps/domain/models.py

```diff
@@ -1016,6 +1019,9 @@ def get_custom_logo(self):
             self.blobs[LOGO_ATTACHMENT].content_type
         )
 
+    def put_attachment(self, *args, **kw):
+        return super(Domain, self).put_attachment(domain=self.name, *args, **kw)
```

Comment:
is this correct? The expected arguments looks to be different for `BlobMixin`, there is content as the first argument?